PWGHF/HFC/Tasks/taskFlow.cxx

```diff
@@ -47,6 +50,8 @@ using namespace o2::framework;
 using namespace o2::framework::expressions;
 
 struct HfTaskFlow {
+  Service<o2::framework::O2DatabasePDG> pdg;
+
```

Comment:
Please order the struct members according to the HF conventions.

Reply:
Same for the other added members.

Reply:
Changed location of that line